aten/src/ATen/native/Resize.cpp

```diff
@@ -21,15 +22,26 @@ Tensor& resize_cpu_(Tensor& self, IntArrayRef size) {
 // this function in native_functions.yaml
 Tensor& resize_as_sparse_(Tensor& self, const Tensor& src);
 
-Tensor& resize_as_(Tensor& self, const Tensor& the_template) {
+Tensor& resize_as_(
+    Tensor& self,
+    const Tensor& the_template,
+    c10::optional<MemoryFormat> optional_memory_format) {
   if (self.is_sparse() && the_template.is_sparse()) {
     return native::resize_as_sparse_(self, the_template);
```

Comment:
It seems weird that we're just ignoring the memory format for sparse. Would it be better to assert that optional_memory_format is nullopt?

Reply:
done